crates/config/src/project/task.rs

```diff
@@ -86,6 +93,9 @@ pub struct TaskOptionsConfig {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub merge_outputs: Option<TaskMergeStrategy>,
 
+    #[serde(rename = "outputStyle", skip_serializing_if = "Option::is_none")]
```

Comment:
Rename isn't necessary since we set it on the struct.

Reply:
Nice, fixed 👍